service_provider.go

```diff
@@ -350,8 +350,8 @@ func (sp *ServiceProvider) ParseResponse(req *http.Request, possibleRequestIDs [
 		if err := xmlsec.Verify(sp.getIDPSigningCert(), rawResponseBuf,
 			xmlsec.SignatureOptions{
 				XMLID: []xmlsec.XMLIDOption{{
-					ElementName:      "Response",
-					ElementNamespace: "urn:oasis:names:tc:SAML:2.0:protocol",
+					ElementName:      "Assertion",
```

Comment:
This change breaks the unit tests, but without it the package doesn't work with OneLogin. Since I'm not a SAML expert, I do not know whether this was wrongly verifying the wrong element or the package needs more complicated logic to figure out what to to verify.
Do you think that I can trust the existing unit tests? The fact that a unit test asserts some output doesn't necessary mean that the output is correct, right? If the existing unit tests are trustworthy, then I will have to read the specs and understand how to extend the package...